blog/tests/admin/test_admin_image_upload_views.py

```
@@ -0,0 +1,113 @@
+):
+    client.login(username="staffuser", password="testpass123")
+
+    url = reverse("blog:admin-image-upload")
+    response = getattr(client, method_name)(url)
+
+    assert response.status_code == 400
+    assert "POST 요청만 허용됩니다." in response.content.decode()
+
+
+@pytest.mark.django_db
+def test_admin_image_upload_rejects_missing_file(client, staff_user):
```

Ruff (0.15.5)

[warning] Unused function argument: `staff_user`

(ARG001)